src/main/java/org/apache/ibatis/jdbc/SqlRunner.java

```diff
@@ -146,16 +139,9 @@ public int insert(String sql, Object... args) throws SQLException {
    * @throws SQLException If statement preparation or execution fails
    */
   public int update(String sql, Object... args) throws SQLException {
-    PreparedStatement ps = connection.prepareStatement(sql);
-    try {
+    try (PreparedStatement ps = connection.prepareStatement(sql)) {
```

Comment:
Probably, this changes will broken a current behavior that ignore exception if the `Statement.close` throw a  `SQLException`.